bitnami/wavefront-adapter-for-istio/templates/deployment.yaml

```diff
@@ -12,6 +12,7 @@ metadata:
   annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
   {{- end }}
 spec:
+  replicas: {{ .Values.replicaCount }}
```

Comment:
Does make sense more than one adapter? if not, we shouldn't allow users increase this number

Reply:
You are right. Taking into account we are deploying the adapter with the wavefront subchart we are always going to want a `1 wavefront deployment = 1 adapter` relation.